priam/src/test/java/com/netflix/priam/config/FakeConfiguration.java

```diff
@@ -227,4 +231,40 @@ public boolean usePrivateIP() {
     public void usePrivateIP(boolean usePrivateIp) {
         this.usePrivateIp = usePrivateIp;
     }
+
+    @Override
+    public boolean skipPrivateIPIngress() {
+        return this.skipPublicIPIngress;
+    }
+
+    public void skipPrivateIPIngress(boolean skipPrivateIPIngress) {
```

Comment:
I would advise to use "set" in the setter functions for 2 reasons:
1. It will give better readability to the user. 
2. It will be in sync with other setter functions already used in this file. 
This comment is also applicable to line 249, 258, 267.

Reply:
Done.